plugins/UFM_NDT_Plugin/ufm_sim_web_service/topo_diff/ndt_infra.py

```diff
@@ -169,6 +170,16 @@ def run_ibdiagnet_verification_command():
     status, cmd_output = execute_generic_command(IBDIAGNET_PORT_VERIFICATION_COMMAND)
     return status
 
+def normalize_port_guid_length(port_guid):
+    '''
+    Normalize port guid len - in need add prefix 0
+    :param port_guid:
+    '''
+    port_guid_len = len(port_guid)
+    if port_guid_len < 18: # in case and we have guid that is short - need to add zero
```

Comment:
please add debug log message for this condition..
please add automation test / unit test to cover this use case.

Reply:
1. No need debug message for such flow - it will be a lot records in log with no any reason.
2. In general tests written by Anas checking this issue on verification setup. Normal flow should work on verification setup. Otherwise we will see errors in log file and incorrect topoconfig file.